blog/releases/announcing-knative-v0-19-release.md

```diff
@@ -0,0 +1,264 @@
+    As `apply` operates on Knative's custom resource types, it suffers the same limitation as the client-side merge algorithm in `kubectl apply` when dealing with CRDs.
+    I.e. arrays can't be merged but are just overwritten by an update.
+    This limitation affects all containers specific parameters.
+
+- For the future, we plan to add a better merge algorithm that supports a full strategic 3-way merge like the support of `kubectl apply` has for intrinsic K8s resources.
+
+- It is important to note that `kn service apply` is fundamentally different to `kn service update` as with `kn service apply` the full configuration needs to be provided as command-line options or within a declaration file when used with `--filename`.
+With `kn service update` you only specify the parts to be updated and don't touch any other service configuration.
+This characteristic of `apply` also implies that you always have to provide the image as the only mandatory parameter with `kn service apply`.
+
+- For example:
+    ```
+    # Initially apply a service with the given image and env var
+    kn service apply random --image rhuss/random:1.0 --env foo=bar
+
+    # Update the service to add a ying=yang env var to the already existing one
+    kn service update random --env ying=yang
+
+    # Apply a full new configuration. Note that foo=bar will be removed
+    # because it is not specified here.
+    kn service apply random --image rhuss/random:1.0 --env ying=yang
+    ```
+
+#### `kn service import`
+
+- `kn service import` is the counterpart to `kn service export` and allows you to import a Service and all active revision that has been exported.
+_Active revisions_ are revisions that are referenced in a traffic-split.
+
+- This command is still marked experimental, and there are some known issues in the import how the `generation` of revision are created.
+Please gives us feedback on how we can improve the support for this feature.
+
+#### Other CLI Features
+
+The following other features have been added, too:
+
+-  An `arm64` flavour has been added to the released artefacts, as well as a `checksum.txt` which caries the sha256sum of all artefacts.
+-  A `channel:` prefix has been added for `--sink` parameters, so that channels can be directly addressed by name.
+-  Aliases are now appearing correctly in the help messages
+- The [Client API](https://github.com/knative/client/blob/b93ca9b34d56621350e43a714a79ba32bd20d7b0/pkg/serving/v1/client.go#L49-L117) as a new list filter `WithLabel()` which can be used when listing services.
+
+### Operator v0.19
+
+TODO
+
+### Thank you contributors v0.19
+
+- [@antoineco](https://github.com/antoineco)
+- [@daisy-ycguo](https://github.com/daisy-ycguo)
+- [@danielhelfand](https://github.com/danielhelfand)
+- [@dprotaso](https://github.com/dprotaso)
+- [@dsimansk](https://github.com/dsimansk)
+- [@eclipselu](https://github.com/eclipselu)
+- [@ian-mi](https://github.com/ian-mi)
```

Comment:
```suggestion
- [@ian-mi](https://github.com/ian-mi)
- [@itsmurugappan](https://github.com/itsmurugappan)
```

Not so obvious from the commits, but @itsmurugappan did quite some contributions. Thanks !

Reply:
I think we should think about to call out contributors more prominently in the client release notes.

Reply:
I have been doing small changes from release notes to release notes, this time I included the list of contributors to give thanks.

Reply:
@rhuss sorry I missed @itsmurugappan I thought I got everyone reading the PRs.

Reply:
Ah I know what happened PR was missing from the changelog https://github.com/knative/client/blob/master/CHANGELOG.adoc#v0190-2020-11-11

That's what I used to extract the github ids for client